### PR TITLE
for review, #2

### PR DESCRIPTION
**Pull request by jmuzerolle**

### CODE REVIEW

File: mos_session/jwebbinar7_nirspecmos.ipynb [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "\n",
```
Comment (ojustino) [inline mos_session/jwebbinar7_nirspecmos.ipynb:18]: <p>Hi @jmuzerolle, before I do the review in full, can you add code for downloading the cached NIRSpec files used in this notebook from Box? The paths in this cell are not accessible to others. (Your JWebbinar 5 notebook has the same issue, but let's just do this notebook for now.)</p><p> </p><p>If you don't remember how, you can use a similar strategy as you did in the "Download Data" section of your JWebbinar 5 notebook. Just substitute in a new Box link and set the symlink to <code>output_dir</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

**Comment by camipacifici** [comment]
@ojustino, I see @jmuzerolle made a couple of commits. Has the review been addressed completely? Is there still something missing? Thank you!

**Comment by ojustino** [comment]
Thanks for looking, @camipacifici. I didn't get a GitHub notification about the new commits. I'm posting an update now.

**Comment by ojustino** [comment]
Hello @jmuzerolle, thank you for the updates. You've resolved all of the style issues but still have a couple of unresolved problems.

## The three-point review (\*action required\*)

1. ⚠️ Notebook execution: **I was unable to run the notebook from top to bottom.**
    - You changed `output_dir` to a path that will only work on your machine (`/Users/muzerol/NIRSpec/Pipeline/Testing/nirspec_files/`). I had previously changed it such that the output folder is created in the user's current directory, regardless of its path. Unless you have objections, please change `output_dir` to `'nirspec_files'`.
    - **Unresolved**: Cells that use `datamodels` to open pipeline products return `ValueError`s about their column names not matching schema. Though the rest of the notebook seemed OK, these cells could not execute fully. Does the same happen for you?

2. ✅ Code style: **There are no remaining PEP8 violations.**

3. ✅ Notebook style: **There are no remaining notebook style issues.**

**Comment by jmuzerolle** [comment]
Hi @ojustino,
Sorry about the output_dir issue, I wanted to put the data in a specific place and intended to change it back before pushing the new version, but forgot. I'll make the correction.
I did not get the datamodels error you are seeing. Perhaps there is a pipeline version issue? There were some changes made I think to the 1D spectra data models since the version used for the webbinar, so that could cause conflicts with the data products on Box. Though I thought I went through the notebook before reprocessing anything and still saw no errors, so I'm not sure.

**Comment by ojustino** [comment]
@jmuzerolle OK, I'll look out for the `output_dir` update.

The other problem could be a pipeline version issue. I just tried creating a new conda environment and am running into a different import error. Which version of `stdatamodels` are you using in the conda environment you created for this notebook?

**Comment by jmuzerolle** [comment]
@ojustino the version of stdatamodels in the environment I used for the original notebook (and to create the pipeline products that are on Box) is 0.2.3.  I'm able to run the notebook without errors using several different environments with different jwst pipeline versions installed - the version used for the webbinar (1.2.3), and the latest development version.

**Comment by jmuzerolle** [comment]
I just pushed a new version with the proper data directory specified.

**Comment by camipacifici** [comment]
@ojustino is this also ready?
Thank you!

**Comment by ojustino** [comment]
@camipacifici, not quite. I've been looking into a dependencies issue.

@jmuzerolle, the problem seems to depend on the versions of `jwst` and `stdatamodels` I have installed:
- `jwst` 1.1.0 + `stdatamodels` 0.3.0 (current default with `pip install -r requirements.txt`): I can't get past the import cells due to a missing module in `stdatamodels`.
- `jwst` 1.1.0 + `stdatamodels` 0.2.4: The import error is resolved, but I see the `ValueError` mentioned in my reviews above.
- `jwst` 1.1.0 + `stdatamodels` 0.2.3: Same as previous.

I don't get any errors when I use `jwst` 1.2.3 with any of the `stdatamodels` versions listed above. **Would it be acceptable to change the version in `requirements.txt` from 1.1.0 to 1.2.3?** If so, let me make the change since I also need to merge in some adjustments from Patrick's review.

**Comment by jmuzerolle** [comment]
@ojustino that should be fine, go ahead and make the change.

**Comment by ojustino** [comment]
Sounds good. My success with `stdatamodels` 0.3.0 ended up being inconsistent, so I added another requirement pointing to an earlier version instead.

@camipacifici, this notebook is clear for a merge now, too.

**Comment by camipacifici** [comment]
Yey, great job everybody!
Merging

**Pull request merged**